doc/mgmt/SONiC_OC_FDB_get_support_HLD.md

```diff
@@ -0,0 +1,271 @@
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+
+sonic# show mac address-table vlan 1
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+
+sonic# show mac address-table interface Ethernet 1
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+sonic#
+
+sonic# show mac address-table interface Ethernet 1
+VlanId  Mac Address             Type            Interface
+1       90:b1:1c:f4:9d:83       dynamic         Ethernet1
+sonic#
+
+sonic# show mac address-table static
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+sonic# show mac address-table static address 00:11:22:33:44:55
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+sonic# show mac address-table static vlan 20
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+
+sonic# show mac address-table static interface Ethernet 1
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+sonic# show mac address-table vlan 20
+VlanId  Mac Address             Type            Interface
+20      00:11:22:33:44:55       static          Ethernet1
+sonic#
+
+````
+#### 3.6.2.3 Debug Commands
+#### 3.6.2.4 IS-CLI Compliance
+The following table maps SONIC CLI commands to corresponding IS-CLI commands. The compliance column identifies how the command comply to the IS-CLI syntax:
+
+- **IS-CLI drop-in replace**  – meaning that it follows exactly the format of a pre-existing IS-CLI command.
+- **IS-CLI-like**  – meaning that the exact format of the IS-CLI command could not be followed, but the command is similar to other commands for IS-CLI (e.g. IS-CLI may not offer the exact option, but the command can be positioned is a similar manner as others for the related feature).
+- **SONIC** - meaning that no IS-CLI-like command could be found, so the command is derived specifically for SONIC.
+
+|CLI Command|Compliance|IS-CLI Command (if applicable)| Link to the web site identifying the IS-CLI command (if applicable)|
+|:---:|:-----------:|:------------------:|-----------------------------------|
+|show mac address-table |IS-CLI drop-in replace | | |
+| show mac address-table count | IS-CLI drop-in replace |  | |
+| show mac address-table address `mac` | IS-CLI drop-in replace |  | |
+| show mac address-table dynamic  |IS-CLI drop-in replace | | |
+| show mac address-table static } |IS-CLI drop-in replace | | |
+| show mac address-table [interface { Ethernet/Port-channel } | IS-CLI drop-in replace | |
+| show mac address-table [vlan ``<id>`` | IS-CLI drop-in replace | |
+
+
+
+### 3.6.3 REST API Support
+#### 3.6.3.1 GET
+##### Get all support for MAC entries
```

Comment:
Can the URIs to be queries also be listed for REST API support

Reply:
Added REST API URIs